cartography/intel/aws/ec2/load_balancer_v2s.py

```diff
@@ -209,6 +365,146 @@ def load_load_balancer_v2_listeners(
     )
 
 
+def rule_conditions_to_strings(conditions: List[Dict]) -> Dict:
+    result = {}
+    for cnd in conditions:
+        if cnd['Field'] == 'http-header':
+            result['http_header'] = f"{cnd['HttpHeaderConfig']['HttpHeaderName']}==[sensitive]"
+        elif cnd['Field'] == 'http-request-method':
+            result['http_request_method'] = '|'.join(cnd['HttpRequestMethodConfig']['Values'])
+        elif cnd['Field'] == 'host-header':
+            result['host_header'] = '|'.join(cnd['HostHeaderConfig']['Values'])
+        elif cnd['Field'] == 'path-pattern':
+            result['path_pattern'] = '|'.join(cnd['PathPatternConfig']['Values'])
+        elif cnd['Field'] == 'query-string':
+            result['query_string'] = '|'.join([f"{k}={v}" for k, v in cnd['QueryStringConfig']['Values'].items()])
+        elif cnd['Field'] == 'source-ip':
+            result['source_ip'] = '|'.join(cnd['SourceIpConfig']['Values'])
+    return result
+
+
+def action_ingest_statement(type: str, additional: str) -> str:
+    statement = """
+    MATCH (rule:ELBV2Rule{id: $RuleId})-[ATTACHED_TO]->(listener:ELBV2Listener)
+    WITH listener, rule
+    UNWIND $Actions as action
+        MERGE (listener)-[rr:ELBV2_LISTENER_RULE]->(actionNode:ELBV2Action:""" + type + """{id: action.id})
```

Comment:
Rather than keep the same information in two places - in the ELBV2Rule _and_ this relationship - I'd rather just keep it in one place and prefer keeping the attributes in the ELBV2Rule node.

There is also an `ATTACHED_TO` relationship between `ELBV2Rule`s and `ELBV2Listeners`. I think having both `ATTACHED_TO` and `ELBV2_LISTENER_RULE` is a bit crowded so we should decide on keeping just one.